pkgs/applications/graphics/inkscape/default.nix

```
@@ -182,14 +168,34 @@ stdenv.mkDerivation rec {
     done
   '';
 
-  passthru.tests.ps2pdf-plugin = callPackage ./test-ps2pdf-plugin.nix { };
+  passthru = {
+    tests = {
+      ps2pdf-plugin = callPackage ./test-ps2pdf-plugin.nix { };
+      inkex = python3Packages.inkex;
+    };
+
+    pythonDependencies = runCommand "python-dependency-list" {
+      nativeBuildInputs = [
+        fd
+        yq
+      ];
+      inherit (finalAttrs) src;
+    } ''
+      unpackPhase
+        tomlq --slurp 'map(.tool.poetry.dependencies | to_entries | map(.key)) | flatten | map(ascii_downcase) | unique' $(fd pyproject.toml) > "$out"
+      '';
```

Sorry, I messed up the indentation:

```suggestion
      tomlq --slurp 'map(.tool.poetry.dependencies | to_entries | map(.key)) | flatten | map(ascii_downcase) | unique' $(fd pyproject.toml) > "$out"
    '';
```

---

No problem. I admit, I also didn't look too closely at the indentation.